src/pykeen/lr_schedulers/__init__.py

```diff
@@ -18,6 +18,7 @@
 __all__.extend((
     subcls.__name__
     for subcls in _LRScheduler.__subclasses__()
+    if subcls.__name__ != 'SWALR'
```

Comment:
😂

Reply:
what is this? a Dr. Seuss joke? Do europeans know what that is?

Reply:
https://pytorch.org/docs/stable/optim.html#stochastic-weight-averaging